packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -1328,44 +1441,100 @@ export class IntervalCollection<TInterval extends ISerializableInterval>
 
 	/**
 	 * {@inheritdoc IIntervalCollection.change}
+	 * @deprecated - call IntervalCollection.change with an object containing the desired parameters
 	 */
-	public change(id: string, start: SequencePlace, end: SequencePlace): TInterval | undefined {
+	public change(id: string, start: SequencePlace, end: SequencePlace);
+
+	public change(id: string, args: ChangeArgs);
+
+	public change(
+		id: string,
+		start: SequencePlace | ChangeArgs,
```

Comment:
nit: I find function body code often slightly more readable if you name these `arg1` / `arg2` and the first step is normalizing the call signature (e.g. into local variables `start, end, properties`)

Reply:
Should I do this for only the implementation signature or for the others as well?

Reply:
Just the implementation, we want the nice names for the public API. The point is to make it obvious that the ordinal arguments aren't meaningful in their own right until we "parse" them to figure out which overload we're in.